### ISSUE TITLE
Disscusion: Change `edit this page` link to main repo

### ISSUE THREAD

**Issue by zzjin** [fanux/sealos-site#44]
Currently, we are using a bot to auto update docs&apis into site repo.
When someone wants to contribute, and click `edit this page` on the website, we need to redirect them to origin sealos repos's doc page.

this can be done by change [docusaurus.config.js](https://github.com/fanux/sealos-site/blob/main/docusaurus.config.js#L39) to
```js
editUrl: "https://github.com/labring/sealos/tree/main/docs/4.0/"
```

But as our site's homepage still not be maintenanced by main repo, here are two choices:

1. Transfer all sites pages to main repo's doc dir.
2. Using custom [EditUrlFunction](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#EditUrlFunction) to spec: docs&api page redirect to main repo, others keep site repo.

Support more contributer's like: #43

**Comment by SignorMercurio** [comment]
Please feel free to open PR and change the config's `editUrl` field. I think this is necessary because it's natural for people to click on these links and contribute without looking at the README.

However, IMO this is the only necessary thing to do for this issue and here's why:

Yes, the site's home page as well as custom pages is maintained here, not the main repo. But the choices listed here are too complex:

1. The home page is a React JS page which depends on a lot of other CSS, JS and image resources. Are we going to migrate them to the main repo too? This doesn't make sense because doing this is no better than migrate this repo entirely to the main repo.
2. This might be workable, but too complex.

I've already stated in README that:

> Please do not change anything under docs and i18n/zh-Hans/docusaurus-plugin-content-docs/current as they are automatically synchronized from [sealos/docs/4.0](https://github.com/labring/sealos/tree/main/docs/4.0). However, if you want to update the [home page](https://github.com/fanux/sealos-site/tree/main/src/pages/index.js) or custom pages like the [company page](https://github.com/fanux/sealos-site/tree/main/src/pages/company.md), feel free to contribute to this repo directly.

And I double-checked the home page and the custom pages and confirmed that there's no `edit this page` button on these pages. Therefore, a contributor who's about to update these pages SHOULD read the README before contributing and found the notes above.

Another reason is that updating these pages, especially home pages, might involve changes to the resources (CSS, JS, images, etc.), and it's more convenient for contributors to test their changes by cloning this repo (or the forked ones) and running `yarn start` to start a local server. They cannot do the same in the cloned main repo.

**Comment by zzjin** [comment]
Fixed by: #48
